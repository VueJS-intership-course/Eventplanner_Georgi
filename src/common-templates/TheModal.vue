<template>
    <div class="back-modal">
        <div class="add-modal container py-3 d-flex justify-content-center">
               <slot></slot>
        </div>
    </div>
</template>

<script setup>
import { onBeforeMount, onBeforeUnmount } from 'vue';

onBeforeMount(() => {
    document.body.style.overflow = 'hidden';
})

onBeforeUnmount(() => {
    document.body.style.overflow = '';
})
</script>

<style lang="scss" scoped>

.back-modal {
    @include modal-style;
};
</style>