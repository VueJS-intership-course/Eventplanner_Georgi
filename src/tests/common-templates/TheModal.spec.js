import TheModal from '@/common-templates/TheModal.vue';
import { mount } from '@vue/test-utils';

const wrapper = mount(TheModal);

it('matches snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
})