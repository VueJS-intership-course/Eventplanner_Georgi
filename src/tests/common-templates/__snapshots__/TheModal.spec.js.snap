// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div class="back-modal">
  <div class="add-modal container py-3 d-flex justify-content-center"></div>
</div>
`;
